cli/azd/pkg/tools/azcli.go

```diff
@@ -148,6 +149,11 @@ type AzCliResource struct {
 	Location string `json:"location"`
 }
 
+type AzCliResourceExtended struct {
+	AzCliResource
+	Kind string `json:"kind"`
+}
```

Comment:
I believe this `Kind` is only used in app service.  Consider renaming struct to `AppServiceResource` or similar.

Reply:
It's standard ARM schema, not specific to AppService: https://docs.microsoft.com/en-us/rest/api/resources/resources/get#genericresource